main/menu.js

```diff
@@ -17,12 +17,32 @@ var dialog = require('./dialog')
 var shell = require('./shell')
 var windows = require('./windows')
 var thumbnail = require('./thumbnail')
+var State = require('../renderer/lib/state')
 
-var menu
+var menu, state
 
 function init () {
   menu = electron.Menu.buildFromTemplate(getMenuTemplate())
   electron.Menu.setApplicationMenu(menu)
+
+  State.load(onState)
```

Comment:
@dcposch I you refer to:
`var config = require('../config')`
We're already loading it in several files.

If you meant `State`, I also felt it wasn't the best approach to load state more than once.
I'd need to spend more time looking into that.
Maybe we can inject it where's needed somehow, or require it using the dispatcher.